WalletWasabi.Fluent/Controls/LabelsItemsPresenter.axaml

```diff
@@ -35,14 +34,18 @@
     <Setter Property="Margin" Value="0" />
     <Setter Property="VirtualizationMode" Value="Simple" />
     <Setter Property="HorizontalAlignment" Value="Left" />
+    <Setter Property="BorderBrush" Value="{DynamicResource TagsBoxBorderBrush}" />
+    <Setter Property="Foreground" Value="{DynamicResource TextControlForeground}" />
+    <Setter Property="MaxLabelWidth" Value="120" />
     <Setter Property="ItemTemplate">
       <DataTemplate>
         <Panel Margin="0 0 4 0">
-          <Border Classes="label" ToolTip.Tip="{Binding}">
+          <Border Classes="label" ToolTip.Tip="{Binding}" BorderBrush="{Binding $parent[c:LabelsItemsPresenter].BorderBrush}">
             <TextBlock Text="{Binding}"
                        TextTrimming="CharacterEllipsis"
                        TextWrapping="NoWrap"
-                       MaxWidth="120" />
```

Comment:
Did you check if CharacterEllipsis works after that changes ?

Reply:
It works. 
![image](https://user-images.githubusercontent.com/3109851/203038174-a7e122dd-8116-4594-9c9c-e103a335ec7c.png)

The value is now set as value in property setter instead of local value.